src/python_package/virtual_pond/__init__.py

```diff
@@ -0,0 +1,174 @@
+"""Virtual pond module"""
+
+import math
+from datetime import datetime, timedelta
+from python_package import rain
+from python_package.rain import area
+
+
+class PondData:  # pylint: disable=R0903
+    """Data from the virtual pond"""
+
+    def __init__(self, height: float, overflow: bool, volume_in: float, volume_out: float):
+        self.height = height
+        self.overflow = overflow
+        self.volume_in = volume_in
+        self.volume_out = volume_out
+
+
+class VirtualPond:  # pylint: disable=R0902
+    """Virtual pond class"""
+
+    def __init__(
+        self,
+        urban_catchment_area_ha: float,
+        surface_reaction_factor: float,
+        discharge_coeficent: float,
+        pond_area_m2: float,
+        water_level_cm: float,
+        water_level_min_cm: float,
+        water_level_max_cm: float,
+        rain_data_mm: rain.Rain,
+    ):  # pylint: disable=R0913
+        self.urban_catchment_area = urban_catchment_area_ha
+        self.surface_reaction_factor = surface_reaction_factor
+        self.discharge_coeficent = discharge_coeficent
+        self.pond_area = pond_area_m2
+        self.water_level = water_level_cm
+        self.water_level_min = water_level_min_cm
+        self.water_level_max = water_level_max_cm
+        self.rain_data = rain_data_mm
+
```

Comment:
intuativt vil jeg mene at pond skulle indeholde koordinaterne for cathment area, så alt info er samlet her.
Det vil sige at defineringen af rain data vil skulle forgå som en opsætnings funktion af pond.

hvis man skal håndtere flere ponds i et større system i fremtiden vil jeg synes at det blev meget forvirrende at man skulle have styr på rain og pond i to forskellige classer frem for at alt ligger under pond